### ISSUE TITLE
empty method should be warning such as int? test() {}

### ISSUE THREAD

**Issue by Silentdoer** [dart-lang/sdk#46293]
because of nullable return type can with no code body, so analyzer should warning no code body method that tip user to implements:
```dart
int? someUtilMethod() {
  // user could forget to implements this method
}

**Comment by eernstg** [comment]
Duplicate of https://github.com/dart-lang/language/issues/1668?

**Comment by Silentdoer** [comment]
> Duplicate of [dart-lang/language#1668](https://github.com/dart-lang/language/issues/1668)?

yes, at least support a default analyzer lint to avoid this code.
